radis/io/hitran.py

```diff
@@ -1594,6 +1596,14 @@ def fetch_hitran(
         load only specific wavenumbers.
     columns: list of str
         list of columns to load. If ``None``, returns all columns in the file.
+    extra_params: 'all' or None
+        Downloads all additional columns available in the HAPI database for the molecule including
+        parameters like `gamma_co2`, `n_co2` that are required to calculate spectrum in co2 diluent.
+        For eg:
+        ```
+        from radis.io.hitran import fetch_hitran
+        df = fetch_hitran('CO', extra_params='all', cache='regen') # cache='regen' to regenerate new database with additional columns
+        ```
```

Comment:
Doctrings are in RST not Markdown. Correction below (you can commit directly) 
```suggestion
        ::
        
            from radis.io.hitran import fetch_hitran
            df = fetch_hitran('CO', extra_params='all', cache='regen') # cache='regen' to regenerate new database with additional columns
        ```
```

Reply:
Okay, but line 1607  containing ` ``` ` should also not be there right?

Reply:
Yes, remove them as well